docs/src/meshes/mesh_data_meshes.md

```diff
@@ -0,0 +1,125 @@
+reference-to-physical mapping can be constructed using only the vertex positions (e.g., an affine
+simplicial mesh or a bilinear quadrilateral mesh). These quantities are generated by most simplicial
+mesh generators, and `StartUpDG.jl` includes both simple uniform meshes via `uniform_mesh`, as
+well as support for triangular meshes constructed using
+[Triangulate.jl](https://github.com/JuliaGeometry/Triangulate.jl), a wrapper around Jonathan Shewchuk's
+[Triangle](https://www.cs.cmu.edu/~quake/triangle.html) package.
+
+The only fields are `md::MeshData`, which contains geometric terms derived from the mapping between
+the reference and physical elements, and `boundary_faces`, which contains a `Dict` of boundary
+segment names (symbols) and list of faces which lie on that boundary segment.
+
+## Variable naming conventions
+
+We use the convention that coordinates on the reference element are ``r`` in 1D, ``r, s`` in 2D,
+or ``r, s, t`` in 3D. Physical coordinates use the standard conventions ``x``, ``x, y``, and
+``x, y, z`` in 1D, 2D, and 3D.
+
+!["Ref-to-physical mapping"](https://user-images.githubusercontent.com/1156048/124361389-a2841380-dbf4-11eb-8ee4-33e71109c8bb.png)
+
+Derivatives of reference coordinates with respect to physical coordinates are abbreviated, e.g.,
+``\frac{\partial r}{\partial x} = r_x``. Additionally, ``J`` is used to denote the determinant of
+the Jacobian of the reference-to-physical mapping.
+
+## Variable meanings and conventions in `StartUpDG.jl`
+
+`StartUpDG.jl` exports structs `RefElemData{NDIMS, ElemShape, ...}` (which contains data associated
+with the reference element, such as interpolation points, quadrature rules, face nodes, normals,
+and differentiation/interpolation/projection matrices) and `MeshData{NDIMS}` (which contains geometric
+data associated with a mesh). These are currently used for evaluating DG formulations in a matrix-free
+fashion. These structs contain fields similar (but not identical) to those in
+`Globals1D, Globals2D, Globals3D` in the Matlab codes from "Nodal Discontinuous Galerkin Methods"
+by Hesthaven and Warburton (2007).
+
+In general, we use the following code conventions:
+* variables such as `r, s,...` and `x, y,...` correspond to values at nodal interpolation points.
+* variables ending in `q` (e.g., `rq, sq,...` and `xq, yq,...`) correspond to values at volume
+quadrature points.
+* variables ending in `f` (e.g., `rf, sf,...` and `xf, yf,...`) correspond to values at face
+quadrature points.
+* variables ending in `p` (e.g., `rp, sp,...`) correspond to "plotting" points, which are usually
+a fine grid of equispaced points.
+* `V` matrices correspond to interpolation matrices from nodal interpolation points, e.g., `Vq`
+interpolates to volume quadrature points, `Vf` interpolates to face quadrature points.
+* geometric quantities in `MeshData` are stored as matrices of dimension
+``\text{number of points per element } \times \text{number of elements}``.
```

Comment:
```suggestion
In general, we use the following code conventions:
* variables such as `r, s,...` and `x, y,...` correspond to values at nodal interpolation points.
* variables ending in `q` (e.g., `rq, sq,...` and `xq, yq,...`) correspond to values at volume
  quadrature points.
* variables ending in `f` (e.g., `rf, sf,...` and `xf, yf,...`) correspond to values at face
  quadrature points.
* variables ending in `p` (e.g., `rp, sp,...`) correspond to "plotting" points, which are usually
  a fine grid of equispaced points.
* `V` matrices correspond to interpolation matrices from nodal interpolation points, e.g., `Vq`
  interpolates to volume quadrature points, `Vf` interpolates to face quadrature points.
* geometric quantities in `MeshData` are stored as matrices of dimension
  ``\text{number of points per element} \times \text{number of elements}``.
```
Otherwise, it doesn't render correctly, see https://trixi-framework.github.io/Trixi.jl/previews/PR690/meshes/mesh_data_meshes/#Variable-meanings-and-conventions-in-StartUpDG.jl

Reply:
Thanks for catching that!